source/simulator/parameters.cc

```diff
@@ -1004,6 +1004,12 @@ namespace aspect
                          "advected with the melt velocity instead of the solid velocity. "
                          "This method can only be chosen if melt transport is active in the "
                          "model."
+                         "\n"
+                         "\\item ``prescribed field'': If a compositional field is marked "
```

Comment:
I am not a big fan of "prescribed field" as this remind me of "prescribed Stokes solution". Not sure if you can think of a better name? If not, we can leave it as is.

Reply:
But it is something very similar to "prescribed Stokes solution". Instead of solving an equation, we use an output of the material model to fix the compositional field to a prescribed value that can also change over time (exactly as in the prescribed Stokes solution). I think that's how we came up with the name. 
The only difference is that for the prescribed Stokes solution, you use a new plugin to prescribe the solution, while here, you use a material model output. But the output doesn't even have to depend on any material properties, you could also just use any function that changes over time, so the functionality is exactly the same.